From 03bc2fe7da8d2cc753cc681be12c8906a9a9a295 Mon Sep 17 00:00:00 2001
From: Ignacio Morelle <shadowm2006@gmail.com>
Date: Thu, 16 Jul 2009 19:30:59 -0400
Subject: [PATCH] IftU: fix a unit filter mistake, which caused some dialogue to be always skipped

---
 scenarios/14_Bye_and_Behold.cfg |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/scenarios/14_Bye_and_Behold.cfg b/scenarios/14_Bye_and_Behold.cfg
index 38d1a0e..67f5f47 100644
--- a/scenarios/14_Bye_and_Behold.cfg
+++ b/scenarios/14_Bye_and_Behold.cfg
@@ -597,7 +597,7 @@ Fire: +30%"
 
         [if]
             [have_unit]
-                role=chatter
+                role=doubtful_elf
             [/have_unit]
             [then]
                 {MSG_UNIT Galas ( _ "We did it. Now what?")}
-- 
1.5.6.5

